Integrations/integration-ArcherRSA.yml

```diff
@@ -2387,46 +2387,85 @@ script:
         })
     }
 
-    function getGroupIDFromGroupName(groupsDataFromArcher, desiredGroupName) {
+    function getGroupIDFromGroupName(groupsDataFromArcher, userSuppliedGroupName) {
         var desiredGroupID = -1;
         for (var i=0; i<groupsDataFromArcher.length; i++){
-            var currentGroupData = groupsDataFromArcher[i].RequestedObject;
-            var currentGroupName = currentGroupData.Name;
-            if (currentGroupName.trim() === desiredGroupName.trim()){
-                desiredGroupID = currentGroupData.Id;
+            var currentArcherGroupData = groupsDataFromArcher[i].RequestedObject;
```

Comment:
are you sure `RequestedObject` will always be in `groupsDataFromArcher[i]` ?

Reply:
Yes, if group data is not null it must have the `RequestedObject` field